src/test_typing_extensions.py

```diff
@@ -5401,6 +5401,18 @@ def test_invalid_uses(self):
             ):
                 Concatenate[1, P]
 
+    @skipUnless(TYPING_3_10_0, "Missing backport to <=3.9. See issue #48")
+    def test_alias_subscription_with_ellipsis(self):
+        P = ParamSpec('P')
+        X = Callable[Concatenate[int, P], Any]
+
+        C1 = X[...]
+        self.assertEqual(C1.__parameters__, ())
+        with self.subTest("Compare Concatenate[int, ...]"):
+            if sys.version_info[:2] == (3, 10):
+                self.skipTest("Needs Issue #110 | PR # 442: construct Concatenate with ...")
```

Comment:
Looks like the new PR this is waiting on is https://github.com/python/typing_extensions/pull/481

```suggestion
                self.skipTest("Needs Issue #110 | PR #481: construct Concatenate with ...")
```